code_schemes/diagnostic_s01e03.json

```diff
@@ -34,170 +34,186 @@
         "Shortcut": "s"
       },
       {
-        "CodeID": "code-3546d695",
+        "CodeID": "code-27b64369",
         "CodeType": "Normal",
         "DisplayText": "about coronavirus",
         "NumericValue": 1,
         "StringValue": "about_coronavirus",
         "VisibleInCoda": true
       },
       {
-        "CodeID": "code-b2139949",
+        "CodeID": "code-d606d1e8",
         "CodeType": "Normal",
         "DisplayText": "symptoms",
         "NumericValue": 2,
         "StringValue": "symptoms",
         "VisibleInCoda": true
       },
       {
-        "CodeID": "code-130abe13",
+        "CodeID": "code-34979c11",
         "CodeType": "Normal",
-        "DisplayText": "spread/transmission",
+        "DisplayText": "how to prevent",
         "NumericValue": 3,
-        "StringValue": "spread_transmission",
+        "StringValue": "how_to_prevent",
         "VisibleInCoda": true
       },
       {
-        "CodeID": "code-0ac18866",
+        "CodeID": "code-6feb376a",
         "CodeType": "Normal",
-        "DisplayText": "how to prevent",
+        "DisplayText": "how to treat",
         "NumericValue": 4,
-        "StringValue": "how_to_prevent",
+        "StringValue": "how_to_treat",
         "VisibleInCoda": true
       },
       {
-        "CodeID": "code-e660f22d",
+        "CodeID": "code-9ec1af3e",
         "CodeType": "Normal",
-        "DisplayText": "how to treat",
+        "DisplayText": "how spread/transmitted",
         "NumericValue": 5,
-        "StringValue": "how_to_treat",
+        "StringValue": "how_spread_transmitted",
         "VisibleInCoda": true
       },
       {
-        "CodeID": "code-5f42bc81",
+        "CodeID": "code-eeb68475",
         "CodeType": "Normal",
-        "DisplayText": "government response",
+        "DisplayText": "what is govt policy",
         "NumericValue": 6,
-        "StringValue": "government_response",
+        "StringValue": "what_is_govt_policy",
         "VisibleInCoda": true
       },
       {
-        "CodeID": "code-8d5a049e",
+        "CodeID": "code-e44f75d6",
         "CodeType": "Normal",
         "DisplayText": "somalia update",
         "NumericValue": 7,
         "StringValue": "somalia_update",
         "VisibleInCoda": true
       },
       {
-        "CodeID": "code-3f99896b",
+        "CodeID": "code-470524f3",
         "CodeType": "Normal",
         "DisplayText": "rumour/stigma/misinfo",
         "NumericValue": 8,
         "StringValue": "rumour_stigma_misinfo",
         "VisibleInCoda": true
       },
       {
-        "CodeID": "code-dd59dae4",
+        "CodeID": "code-5dba1303",
         "CodeType": "Normal",
-        "DisplayText": "collective hope",
+        "DisplayText": "government responce",
         "NumericValue": 9,
+        "StringValue": "government_responce",
```

Comment:
```suggestion
        "StringValue": "government_response",
```

Reply:
keeping as-is to match the previous diagnostic dataset.